src/main/kotlin/com/sourcegraph/cody/commands/CommandId.kt

```diff
@@ -2,8 +2,8 @@ package com.sourcegraph.cody.commands
 
 import java.awt.event.KeyEvent
 
-enum class CommandId(val displayName: String, val mnemonic: Int) {
-  Explain("Explain Code", KeyEvent.VK_E),
-  Smell("Smell Code", KeyEvent.VK_S),
-  Test("Generate Test", KeyEvent.VK_T)
+enum class CommandId(val id: String, val displayName: String, val mnemonic: Int) {
```

Comment:
The class is `CommandId` and it has `id` field now. That leads to `commandId.id` which looks weird. Can we remove `Id` from the class name?

Reply:
Let me do this either on a separate PR or at the end of the reviews... it is used in a quite a few places, I do not want to introduced such amount of noise on this PR.

Reply:
I think this is acceptable. In fact, such naming seems to be [rather common](https://github.com/search?q=Id%28val+id%3A&type=code) in Kotlin and Scala code.